packages/eslint-plugin-xcritical/package.json

```diff
@@ -46,14 +46,11 @@
 		"babel-eslint": "10.1.0",
 		"eslint": "6.8.0",
 		"eslint-config-airbnb": "18.1.0",
-		"eslint-config-prettier": "6.11.0",
```

Comment:
This ones were unused?

- eslint-config-prettier
- eslint-plugin-prettier
- prettier

Reply:
Yep